libraries/engage/product/components/Swatch/index.jsx

```diff
@@ -0,0 +1 @@
+export { Swatch } from './Swatch';
```

Comment:
I dont see benefits to have intermediate export file. Its is already exported in `product` level

Reply:
This makes importing more intuitive. When you import this within another engage component, you can't use the global index file because of unit testing.